src/Discord.Net.Rest/Entities/Channels/RestVoiceChannel.cs

```diff
@@ -9,10 +9,11 @@
 namespace Discord.Rest
 {
     [DebuggerDisplay(@"{DebuggerDisplay,nq}")]
-    public class RestVoiceChannel : RestGuildChannel, IVoiceChannel, IRestAudioChannel
+    public class RestVoiceChannel : RestGuildChannel, IVoiceChannel, IRestAudioChannel, INestedChannel
```

Comment:
Redundant inheritance since `IVoiceChannel` should has it.
(Forgot to make `IVoiceChannel` inherit `INestedChannel`)